test/runtestcases.bat

```diff
@@ -0,0 +1,25 @@
+@ECHO OFF
+
+SET cmd=%1
+SET dir=%2
+
+echo %dir%
+
+FOR /f "tokens=*" %%G IN ('dir /b %dir%\*.t') DO (
+  ECHO "%dir%\%%G"
+  CALL %cmd% "%dir%\%%G"
+
+  IF %ERRORLEVEL% EQU 0 (
```

Comment:
This doesn't seem to work (it always writes PASSED). I think you need to use !ERRORLEVEL! to request delayed expansion.

Reply:
thanks for the remark!